test/run-integration.js

```diff
@@ -330,13 +331,33 @@ function dnsFailThenSuccess(t) {
   });
 }
 
+function dnsFailThenSuccessNoSearch(t) {
+  t.plan(1);
+
+  var client_ = algoliasearch(
+    appId,
+    apiKey, {
+      // .biz is a black hole DNS name (not resolving)
+      hosts: [appId + '-dsn.algolia.biz', appId + '-dsn.algolia.net'],
+      timeout: 120000, // let's wait for the DNS timeout
```

Comment:
A minor one, could we constify this timeout e.g
```javascript
const dnsTimeout = 120000;
```
and then use it in the 3 places that `120000` is being used? Am asking for this as I noticed that the value `40000` was changed to `120000` suggesting that the timeout was realized to previously be short.